packages/bpk-component-banner-alert/docs/migrating-from-v1-to-v2.md

```diff
@@ -0,0 +1,120 @@
+
+### The default case
+
+No `dismissable` prop and no children.
+
+**V1:**
+
+```javascript
+import BpkBannerAlert, { ALERT_TYPES } from "bpk-component-banner-alert";
+
+const render = () => (
+  <BpkBannerAlert type={ALERT_TYPES.SUCCESS} message="Everything went well" />
+);
+```
+
+This migration is really simple as you have to make no changes unless you have using `ariaLive` which has been removed.
+
+**V2:**
+
+```javascript
+import BpkBannerAlert, { ALERT_TYPES } from "bpk-component-banner-alert";
+
+const render = () => (
+  <BpkBannerAlert type={ALERT_TYPES.SUCCESS} message="Everything went well" />
+);
+```
+
+## The dismissable case
+
+In the dismissable case the component needs to be changed to `BpkBannerAlertDismissable`. Also verify that you aren't passing props that `BpkBannerAlertDismissable` doesn't use.
+
+
+**V1:**
+
+```javascript
+import BpkBannerAlert, { ALERT_TYPES } from "bpk-component-banner-alert";
+
+const render = () => (
+  <BpkBannerAlert
+    dismissable
+    dismissButtonLabel="Dismiss"
+    type={ALERT_TYPES.SUCCESS}
+    message="Everything went well"
+    onDismiss={() => undefined}
+  />
+);
+```
+
+**V2:**
+
+```javascript
+import {
+  ALERT_TYPES,
+  BpkBannerAlertDismissable
+} from "bpk-component-banner-alert";
+
+const render = () => (
+  <BpkBannerAlertDismissable
+    dismissButtonLabel="Dismiss"
+    type={ALERT_TYPES.SUCCESS}
+    message="Everything went well"
+    onDismiss={() => undefined}
+  />
+);
+```
+
+## The expandable case
+
+In version 1 the expandable behaviour was enabled if the component had any children. With version 2 there is now a dedicated component, `BpkBannertAlertExpandable`, for this bevaiour.
+
+**V1:**
+
+```javascript
+import BpkBannerAlert, { ALERT_TYPES } from "bpk-component-banner-alert";
+
+const render = () => (
+  <BpkBannerAlert
+    type={ALERT_TYPES.SUCCESS}
+    message="Everything went horribly wrong"
+    toggleButtonLabel="Toggle details"
+    onExpandToggle={() => undefined}
+  >
+    An even longer more detailed message
+  </BpkBannerAlert>
+);
+```
+
+**V2:**
+
+```javascript
+import { ALERT_TYPES, BpkBannerAlertExpandable } from "bpk-component-banner-alert";
+
+const render = () => (
+  <BpkBannerAlertExpandable
+    type={ALERT_TYPES.SUCCESS}
+    message="Everything went horribly wrong"
+    toggleButtonLabel="Toggle details"
+    onExpandToggle={() => undefined}
+  >
+    An even longer more detailed message
+  </BpkBannerAlertExpandable>
+);
+```
+
+## `withBannerAlertState`
+
+`withBannerAlertState` continues to work with both `BpkBannerAlertDismissable` and `BpkBannerAlertExpandable` as before. However when using `hideAfter` `onDimiss` is no longer called when the banner alert is removed, instead a new prop `onHide` has been introduced for this purposeas before. However when using `hideAfter` `onDimiss` is no longer called when the banner alert is removed, instead a new prop `onHide` has been introduced for this purpose.
```

Comment:
Looks like the same sentence (beginning `However when using...`) is repeated here.